interpreter/CMakeLists.txt

```diff
@@ -580,3 +580,35 @@ mark_as_advanced(FORCE BUG_REPORT_URL BUILD_CLANG_FORMAT_VS_PLUGIN BUILD_SHARED_
                        C_INCLUDE_DIRS DEFAULT_SYSROOT FFI_INCLUDE_DIR FFI_LIBRARY_DIR
                        GCC_INSTALL_PREFIX LIBCLANG_BUILD_STATIC TOOL_INFO_PLIST)
 mark_as_advanced(CLEAR LLVM_ENABLE_ASSERTIONS LLVM_BUILD_TYPE)
+
+if(NOT LLVM_TABLEGEN_EXE)
+    message(STATUS "llvm-tblgen not found. Attempting to build llvm-tblgen.")
```

Comment:
Is this necessary? With `builtin_llvm` we should have already built it. For external LLVM packages, are there installations of the development headers but without `llvm-tablegen`?

Reply:
That's right, thanks. I tried it initially and the build was failing for me. I realized it was just that I was looking for `LLVM_TABLEGEN_EXE` at the wrong place.